src/server/dfly_main.cc

```diff
@@ -594,7 +595,20 @@ void sigill_hdlr(int signo) {
   exit(1);
 }
 
+void print_basic_usage_info() {
+  std::cout << "* Default logdir is: \"/tmp/\". It can be overwritten with flag --log_dir=[path]\n";
```

Comment:
@romange I am not a big fun of this, I tried to use `LOG(INFO)` && `absl::SetFlag(&FLAGS_logtostdout, true);` but it still prints file related information which (agreeing with the discord user) is `dev` related info. Therefore I opted in to use `cout` to make it easier to print the formatted code.

Moreover, I agree with the general ideas mentioned on the discord conversation and for that I would suggest for us to think the format and layout and the information we would like to print on startup in the future(to be closer to the user-friendly spectrum than dev related info). That being said, I opted in to print "unformated" lines , the exact layout and format of our startup info should be discussed separately from this PR (unless ofc you strongly object with the aforementioned).

Reply:
Nit: I'd use the new-ish raw string literal syntax, i.e:

```cpp
std::cout << r"(* Default logdir is: "/tmp/". It can be overwritten with flag --log_dir=[path]
* Logfile name format is:
...
```

Reply:
@chakaz LIT suggestion